src/test/java/org/folio/rest/api/ItemDamagedStatusAPITest.java

```diff
@@ -0,0 +1,247 @@
+package org.folio.rest.api;
+
+import static org.apache.commons.lang3.StringUtils.EMPTY;
+import static org.folio.rest.support.http.InterfaceUrls.itemDamagedStatusesUrl;
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.CoreMatchers.notNullValue;
+import static org.junit.Assert.assertThat;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.UUID;
+
+import org.folio.HttpStatus;
+import org.folio.rest.impl.ItemDamagedStatusAPI;
+import org.folio.rest.jaxrs.model.ItemDamageStatus;
+import org.junit.Before;
+import org.junit.Test;
+
+import io.vertx.core.json.JsonObject;
+
+public class ItemDamagedStatusAPITest extends TestBase {
+
+  public static final String TEST_TENANT = "test_tenant";
+
+  @Before
+  public void beforeEach() {
+    StorageTestSuite.deleteAll(TEST_TENANT, ItemDamagedStatusAPI.REFERENCE_TABLE);
+  }
+
+  @Test
+  public void canCreateItemDamagedStatus() {
+
```

Comment:
The other tests do not have a leading blank line, I believe this can be removed

Reply:
Done